utils/flink-components-testkit/src/main/scala/pl/touk/nussknacker/engine/flink/util/test/FlinkTestScenarioRunner.scala

```diff
@@ -142,6 +142,17 @@ class FlinkTestScenarioRunner(
     }
   }
 
+  /**
+   * Can be used to test Batch scenarios based on expressions where input doesn't matter. In such case, the input should
+   * be treated only as a single trigger of the flow of data.
+   */
+  def runWithSingleRecordBounded[R](scenario: CanonicalProcess): RunnerListResult[R] =
```

Comment:
Don't know if adding something like this to utils is ok? I can limit it to an extension in batch package, but it will be useful in the future.

Reply:
Let's try to use `runWithData` with `List(1)` and change `boundedness` to `Boundedness.BOUNDED` there.